security/hashcat/Portfile

```diff
@@ -1,19 +1,10 @@
 # -*- coding: utf-8; mode: _tcl; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- vim:fenc=utf-8:ft=tcl:et:sw=2:ts=2:sts=2
 
 PortSystem              1.0
+PortGroup               github 1.0
 
```

Comment:
please add the `makefile 1.0` PG as well so that the MacPorts settings are propagated correctly. For example, right now the port is not using TheRightCompiler. Additionally, when using this PG you can remove a few lines later on in the Portfile; see below).

Reply:
OK, change has been made; `makefile` portgroup added.